hack/testing/init-log-stack-ansible

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+OS_ANSIBLE_DIR=${OS_ANSIBLE_VERSION:-$OS_O_A_L_DIR/lib/openshift-ansible}
+OS_PLAYBOOK_VERBOCITY=${OS_PLAYBOOK_VERBOCITY:-"vv"}
+
+## setup ansible
+os::cmd::expect_success "sudo yum makecache fast"
+os::cmd::expect_success "sudo yum install python-devel ansible python2-pip -y"
+os::cmd::expect_success "sudo pip install ruamel.yaml"
+
+# make sure binaries are in the desired place
+# hack
+for oc_binary in oc oadm
+do
+  if [ ! -f /usr/local/bin/$oc_binary ]; then
+    echo "Copying $oc_binary from path to /usr/local/bin for use by openshift-ansible"
+    os::cmd::expect_success "sudo cp `which $oc_binary` /usr/local/bin"
+  else
+    echo "Found $oc_binary in /usr/local/bin for use by openshift-ansible"
+  fi
+  if [ ! -f /usr/bin/$oc_binary ]; then
+    echo "Copying $oc_binary from path to /usr/bin for use by openshift-ansible"
+    os::cmd::expect_success "sudo cp `which $oc_binary` /usr/bin"
+  fi
+done
+
+export PATH=/usr/local/bin:$PATH
+
+source $OS_O_A_L_DIR/hack/testing/build-images
+
+# init inventory file
+pushd $OS_ANSIBLE_DIR
+
+# these should be interpreted as python
+cat > ansible.inventory  << EOL
+[oo_first_master]
+openshift
+
+[oo_first_master:vars]
+ansible_become=true
+ansible_connection=local
+containerized=true
+docker_protect_installed_version=true
+deployment_type=origin
+required_packages=[]
+
+openshift_logging_image_prefix=$imageprefix
+openshift_logging_use_ops=${ENABLE_OPS_CLUSTER:-False}
+openshift_logging_fluentd_use_journal=${USE_JOURNAL:-False}
+openshift_logging_fluentd_journal_read_from_head=${JOURNAL_READ_FROM_HEAD:-False}
+
+EOL
+
+echo "### Created host inventory file ###"
+cat ansible.inventory
+echo "###################################"
+
+# Execute logging playbook
+os::cmd::expect_success "oc login --username=system:admin"
+
+playbook_cmd="sudo ansible-playbook -i ansible.inventory \
+  -$OS_PLAYBOOK_VERBOCITY \
+  playbooks/common/openshift-cluster/openshift_logging.yml \
+  -e ansible_host=127.0.0.1 \
+  -e openshift_master_config_dir=${SERVER_CONFIG_DIR}/master"
```

Comment:
You want to use ansible to install on the local system?  You might also add `-c local`